manifests/install.pp

```diff
@@ -3,7 +3,7 @@
   $ensure  = 'latest',
   ) {
 
-  package { 'scout':
+  package { '^scout':
```

Comment:
We don't want to install a gem called `^scout`.

Where does the version check occur?